smartsim/_core/launcher/step/mpi_step.py

```diff
@@ -133,29 +135,31 @@ def _build_exe(self) -> t.List[str]:
         if self._get_mpmd():
             return self._make_mpmd()
 
-        exe = self.run_settings.exe
-        args = self.run_settings._exe_args  # pylint: disable=protected-access
+        exe = self.entity.exe
+        args = self.entity.exe_args  # pylint: disable=protected-access
```

Comment:
Nit, no longer accessing a protected attr
```suggestion
        args = self.entity.exe_args
```

Reply:
@juliaputko i think you'll fix this

Reply:
To be resolved in [this ticket](https://github.com/CrayLabs/SmartSim/issues/762)

Reply:
fixed already